ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/AttestationUtil.java

```diff
@@ -61,8 +63,8 @@ private static boolean isAttestationCurrentEpoch(
     // TODO: Replace longValue wiht UnsignedLong
     long epoch_lower_boundary = state.getSlot().longValue() - Constants.EPOCH_LENGTH;
     long epoch_upper_boundary = state.getSlot().longValue();
-    return (record.getData().getSlot() <= epoch_lower_boundary
-        && record.getData().getSlot() > epoch_upper_boundary);
+    return (record.getData().getSlot().intValue() <= epoch_lower_boundary
```

Comment:
Comparing an UnsignedLong should only be done using UnsignedLong's compareTo value. In addition, using intValue is dangerous unless we're positive `record.getData().getSlot().intValue()` is bounded by 2^31-1, which we're not given the nature of a slot.